Chore: update the latest version to 1.4.3 (#769)
oam-dev/kubevela.io: docs/install.mdx

Addon: velaux enabled Successfully.
```

> By default, velaux didn't have any exposed port, If installing it locally, you can view it by:

Reviewer: ```suggestion
By default, velaux didn't have any exposed port, you can view it by:
```

Reviewer: 非本地也能运行

Author: 这个比较加前提条件，否则走不通。在远程主机上执行这个命令也访问不到。

Reviewer: 可以啊，只有本地的 vela 能连到 k8s 集群

```
vela port-forward addon-velaux -n vela-system 8080:80
```

Choose `> Cluster: local | Namespace: vela-system | Component: velaux | Kind: Service` for visit.

> If you installing it in remote node, please refer to [VelaUX addon docs](./reference/addons/velaux) for more advanced installation ways.

Reviewer: ```suggestion
If you are installing it in a remote environment such as a virtual machine, you can refer to [VelaUX addon](./reference/addons/velaux) document to expose an endpoint or other advanced installation arguments.
```

> VelaUX needs authentication. The default username is `admin` and the password is `VelaUX12345`.

Reviewer: ```suggestion
VelaUX needs authentication. The default username is **`admin`** and the password is **`VelaUX12345`**.
```
